monkestation/code/datums/station_traits/neutral_traits.dm

```diff
@@ -12,3 +12,20 @@
 /datum/station_trait/announcement_duke/New()
 	. = ..()
 	SSstation.announcer = /datum/centcom_announcer/duke
+
+/datum/station_trait/new_space_law
+	name = "New Space Law"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 10
+	show_in_report = TRUE
+	report_message = "A recent bet at centcom has led to a new space law being created. A report on it will be printed at all security record consoles."
+
+/datum/station_trait/new_space_law/on_round_start()
+	var/new_space_law = pick(world.file2list("monkestation/strings/new_space_laws.txt"))
+	priority_announce("Due to recent events in space politics [new_space_law] now 1xx illegal under space law.")
```

Comment:
Contradicts the report message and has poor grammar.

Reply:
The grammar is inside the new spaces laws strings for variability as some things use "is" and some things use "are".
The CC report text is what command is told and the general message is what the crew is told,(from a "lore" perspective) but I can change that if needed.